packages/harpguru-core/src/utils/get-menu-styles-and-animation-vals/get-menu-styles-and-animation-vals.ts

```diff
@@ -142,12 +130,12 @@ export const getMenuStylesAndAnimationVals = (
   })
 
   return {
-    styles,
     menuSlideXTranslation,
     menuSlideYTranslation,
     menuScale,
     menuBackgroundColor,
     menuOpacity,
     labelCounterScale,
+    labelProtrusion,
   }
 }
```

Comment:
This whole file deserves a review along with the existing location of some of it's dependent static value dependencies.

Reply:
Done